docs/proposals/tinyms-support.md

```diff
@@ -0,0 +1,139 @@
+#  Implement Sedna Python SDK support TinyMS
+# 1. Project Background
+## 1.1 Project Background
+
+Sedna is an edge-cloud synergy AI project incubated in KubeEdge SIG AI. Benefiting from the edge-cloud synergy capabilities provided by KubeEdge, Sedna can implement across edge-cloud collaborative training and collaborative inference capabilities, such as joint inference, incremental learning, federated learning, and lifelong learning. Sedna supports popular AI frameworks, such as TensorFlow, Pytorch, PaddlePaddle, and MindSpore.
+
+Sedna can simply enable edge-cloud synergy capabilities to existing training and inference scripts, bringing the benefits of reducing costs, improving model performance, and protecting data privacy.
+
+## 1.2 Project Pain Points
+
+Currently, Sedna only supports sklearn's xgboost, Tensorflow, Keras and MindSpore, but doesn’t support MindSpore’s high-level API tool TinyMS.
+
+This project aims to implement Senda Python SDK ‘s support for TinyMS based on  Sedna's lib development interface and implements an Incremental Learning demo.
+
+## 1.3 Project Goals
```

Comment:
The benifit of TinyMS integration should be included in this section, not the target of OSPP.